testCommon/src/test/resources/setup-with-single-remote.sh

```diff
@@ -44,8 +49,11 @@ cd machete-sandbox
     push
   create_branch hotfix/add-trigger
     commit HOTFIX Add the trigger
+    commit HOTFIX Add the trigger - fixes
     push
-    git commit --amend -m 'HOTFIX Add the trigger (amended)'
```

Comment:
instead of diverged and newer let's have diverged and older case - that will enable reset to remote action in the toolbar. 

the action however is available from the context-menu - we are a bit inconsistent b/c of https://github.com/VirtusLab/git-machete-intellij-plugin/issues/427. 

in general I am calling current branch referring actions from toolbar and non-current from context-menu to cover both cases